docs/source/app-dev/grpc/index.rst

```diff
@@ -66,19 +73,31 @@ For information on how Daml types and contracts are represented by the Ledger AP
 Error handling
 **************
 
-For the standard error codes that the server or the client might return, see the `gRPC documentation <https://github.com/grpc/grpc/blob/600272c826b48420084c2ff76dfb0d34324ec296/doc/statuscodes.md>`__ .
+The Ledger API generally uses the gRPC standard error codes for signaling response failures to client applications.
+
+For more details on the gRPC standard error codes, see the `gRPC documentation <https://github.com/grpc/grpc/blob/600272c826b48420084c2ff76dfb0d34324ec296/doc/statuscodes.md>`__ .
 
-For submitted commands, there are these response codes:
+Generically, on submitted commands the Ledger API responds with the following gRPC status codes:
```

Comment:
Could this section be dropped and instead we should point to the error categories inventory?